examples/step-32/doc/intro.dox

```diff
@@ -1144,7 +1144,7 @@ the following quantities:
   viscosity of the material that flows into the area vacated under the
   rebounding continental plates.
 
-  Using this technique, values around $\eta=10^{21} \text{Pa \thinmuskip s}
+  Using this technique, values around $\eta=10^{21} \text{Pa \; s}
```

Comment:
Bah. My copy of MathJax put a `\;` in the output. Let me take a closer look.

Reply:
I think mathjax can handle it, but if you don't have mathjax enabled, compilation fails.

Reply:
and no, it looks wrong at https://www.dealii.org/developer/doxygen/deal.II/step_32.html

Reply:
I still see `\;`s on that web page.

Reply:
(not that my 'fix' did any better...)

Reply:
no, your \thinmuskip breaks the rest of the formula. It is fine with this PR.